addons/library.kodi.guilib/libKODI_guilib.h

```diff
@@ -825,13 +825,13 @@ class CAddonGUIRenderingControl
   virtual ~CAddonGUIRenderingControl();
   virtual void Init();
 
-  virtual bool Create(int x, int y, int w, int h, void *device);
+  virtual bool Create(int x, int y, int w, int h, float ratio);
```

Comment:
ahh, I see. This is a left over and should be changed the other way round

1)
device can be reset, i.e. move window from one monitor to another
3)
vnsi and other addons that use GUIRenderingControl are no viz.